### PR TITLE
Presentation: Fix `BisCore` supplemental rules for related `bis.ExternalSource` properties

### PR DESCRIPTION
**Pull request by grigasp**
Fixes https://github.com/iTwin/itwinjs-core/issues/5374

### CODE REVIEW

**Comment by grigasp** [comment]
@mergifyio backport release/3.7.x

**Review by saskliutas — APPROVED** [review]

File: full-stack-tests/presentation/src/Utils.ts [hunk 1]
```diff
@@ -67,17 +67,34 @@ export function getFieldsByLabel(rootFields: Field[], label: string): Field[] {
   return foundFields;
 }
 
-export async function buildTestIModel(name: string, cb: (builder: IModelBuilder) => void): Promise<IModelConnection> {
-  const outputFile = setupOutputFileLocation(name);
+function createValidIModelFileName(imodelName: string) {
+  return sanitize(imodelName.replace(" ", "-")).toLocaleLowerCase();
```
Comment (pmconne) [inline full-stack-tests/presentation/src/Utils.ts:71]: Did we have tests that were producing invalid file names?

**Review by pmconne — APPROVED** [review]

File: full-stack-tests/presentation/src/Utils.ts (hunk 1 above)
Comment (grigasp) [reply full-stack-tests/presentation/src/Utils.ts:71]: yes, for example this one: `removes 'Source Information -> Model Source' properties`

Comment (grigasp) [reply full-stack-tests/presentation/src/Utils.ts:71]: I'm using full test name as the name of the imodel it generates

**Pull request merged**